coralproject/talk
# [CORL-1332] By default load 10 replies before showing expand options

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- make a comment
- reply to it 10 or more times
- refresh page
- see comment
- see 10 replies
